{"nodes":[{"pos":[876,969],"content":"The exception that is thrown for a general XAML reader or XAML writer exception. See Remarks.","needQuote":true,"needEscape":true,"nodes":[{"content":"The exception that is thrown for a general XAML reader or XAML writer exception. See Remarks.","pos":[0,93],"nodes":[{"content":"The exception that is thrown for a general XAML reader or XAML writer exception.","pos":[0,80]},{"content":"See Remarks.","pos":[81,93]}]}]},{"pos":[981,1260],"content":"More specific exceptions such as <xref:System.Xaml.XamlParseException> exist and are often thrown instead of XamlException. Specific exceptions typically provide more precise exception information as properties of the exception, or just because the specific exception was raised.","needQuote":true,"needEscape":true,"extradata":"MT","nodes":[{"content":"More specific exceptions such as <xref:System.Xaml.XamlParseException> exist and are often thrown instead of XamlException. Specific exceptions typically provide more precise exception information as properties of the exception, or just because the specific exception was raised.","pos":[0,279],"nodes":[{"content":"More specific exceptions such as &lt;xref:System.Xaml.XamlParseException&gt; exist and are often thrown instead of XamlException.","pos":[0,123],"source":"More specific exceptions such as <xref:System.Xaml.XamlParseException> exist and are often thrown instead of XamlException."},{"content":"Specific exceptions typically provide more precise exception information as properties of the exception, or just because the specific exception was raised.","pos":[124,279]}]}]},{"pos":[1743,1904],"content":"Initializes a new instance of the <xref href=\"System.Xaml.XamlException\"></xref> class. The instance contains a system-supplied message that describes the error.","needQuote":true,"needEscape":true,"nodes":[{"content":"Initializes a new instance of the <bpt id=\"p1\">&lt;xref href=\"System.Xaml.XamlException\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> class.","pos":[0,87],"source":"Initializes a new instance of the <xref href=\"System.Xaml.XamlException\"></xref> class."},{"content":"The instance contains a system-supplied message that describes the error.","pos":[88,161]}]},{"pos":[2424,2579],"content":"Initializes a new instance of the <xref href=\"System.Xaml.XamlException\"></xref> class. The instance contains a specified message that describes the error.","needQuote":true,"needEscape":true,"nodes":[{"content":"Initializes a new instance of the <bpt id=\"p1\">&lt;xref href=\"System.Xaml.XamlException\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> class.","pos":[0,87],"source":"Initializes a new instance of the <xref href=\"System.Xaml.XamlException\"></xref> class."},{"content":"The instance contains a specified message that describes the error.","pos":[88,155]}]},{"pos":[2721,2876],"content":"The message that describes the exception. The caller of this constructor must make sure that this string has been localized for the current system culture.","needQuote":true,"needEscape":true,"nodes":[{"content":"The message that describes the exception. The caller of this constructor must make sure that this string has been localized for the current system culture.","pos":[0,155],"nodes":[{"content":"The message that describes the exception.","pos":[0,41]},{"content":"The caller of this constructor must make sure that this string has been localized for the current system culture.","pos":[42,155]}]}]},{"pos":[3571,3679],"content":"Initializes a new instance of the <bpt id=\"p1\">&lt;xref href=\"System.Xaml.XamlException\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> class with serialized data.","needQuote":true,"needEscape":true,"source":"Initializes a new instance of the <xref href=\"System.Xaml.XamlException\"></xref> class with serialized data."},{"pos":[3946,3995],"content":"The object that holds the serialized object data.","needQuote":true,"needEscape":true,"nodes":[{"content":"The object that holds the serialized object data.","pos":[0,49]}]},{"pos":[4091,4150],"content":"The contextual information about the source or destination.","needQuote":true,"needEscape":true,"nodes":[{"content":"The contextual information about the source or destination.","pos":[0,59]}]},{"pos":[4312,4394],"content":"<ph id=\"ph1\">&lt;code&gt;info&lt;/code&gt;</ph> is <bpt id=\"p1\">&lt;xref uid=\"langword_csharp_null\" name=\"null\" href=\"\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept>.","needQuote":true,"needEscape":true,"source":"<code>info</code> is <xref uid=\"langword_csharp_null\" name=\"null\" href=\"\"></xref>."},{"pos":[4849,5060],"content":"Initializes a new instance of the <xref href=\"System.Xaml.XamlException\"></xref> class. The instance contains a specified error message and a reference to the inner exception that is the cause of this exception.","needQuote":true,"needEscape":true,"nodes":[{"content":"Initializes a new instance of the <bpt id=\"p1\">&lt;xref href=\"System.Xaml.XamlException\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> class.","pos":[0,87],"source":"Initializes a new instance of the <xref href=\"System.Xaml.XamlException\"></xref> class."},{"content":"The instance contains a specified error message and a reference to the inner exception that is the cause of this exception.","pos":[88,211]}]},{"pos":[5072,5496],"content":"An exception that is thrown because a previous exception should include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property. The <xref:System.Exception.InnerException%2A> property returns the same value that is passed into the constructor; or it returns `null`if the <xref:System.Exception.InnerException%2A> property does not provide the inner exception value to the constructor.","needQuote":true,"needEscape":true,"extradata":"MT","nodes":[{"content":"An exception that is thrown because a previous exception should include a reference to the previous exception in the &lt;xref:System.Exception.InnerException%2A&gt; property.","pos":[0,168],"source":"An exception that is thrown because a previous exception should include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property."},{"content":"The &lt;xref:System.Exception.InnerException%2A&gt; property returns the same value that is passed into the constructor; or it returns <ph id=\"ph1\">`null`</ph>if the &lt;xref:System.Exception.InnerException%2A&gt; property does not provide the inner exception value to the constructor.","pos":[169,424],"source":" The <xref:System.Exception.InnerException%2A> property returns the same value that is passed into the constructor; or it returns `null`if the <xref:System.Exception.InnerException%2A> property does not provide the inner exception value to the constructor."}]},{"pos":[5664,5819],"content":"The message that describes the exception. The caller of this constructor must make sure that this string has been localized for the current system culture.","needQuote":true,"needEscape":true,"nodes":[{"content":"The message that describes the exception. The caller of this constructor must make sure that this string has been localized for the current system culture.","pos":[0,155],"nodes":[{"content":"The message that describes the exception.","pos":[0,41]},{"content":"The caller of this constructor must make sure that this string has been localized for the current system culture.","pos":[42,155]}]}]},{"pos":[5893,6193],"content":"The exception that is the cause of the current exception. If the `innerException` parameter is not <xref uid=\"langword_csharp_null\" name=\"null\" href=\"\"></xref>, the current exception is raised in a <xref uid=\"langword_csharp_catch\" name=\"catch\" href=\"\"></xref> block that handles the inner exception.","needQuote":true,"needEscape":true,"nodes":[{"content":"The exception that is the cause of the current exception.","pos":[0,57]},{"content":"If the <ph id=\"ph1\">`innerException`</ph> parameter is not <bpt id=\"p1\">&lt;xref uid=\"langword_csharp_null\" name=\"null\" href=\"\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept>, the current exception is raised in a <bpt id=\"p2\">&lt;xref uid=\"langword_csharp_catch\" name=\"catch\" href=\"\"&gt;</bpt><ept id=\"p2\">&lt;/xref&gt;</ept> block that handles the inner exception.","pos":[58,300],"source":" If the `innerException` parameter is not <xref uid=\"langword_csharp_null\" name=\"null\" href=\"\"></xref>, the current exception is raised in a <xref uid=\"langword_csharp_catch\" name=\"catch\" href=\"\"></xref> block that handles the inner exception."}]},{"pos":[6798,6973],"content":"Initializes a new instance of the <xref href=\"System.Xaml.XamlException\"></xref> class. The instance contains a specified error message, inner exception, and line information.","needQuote":true,"needEscape":true,"nodes":[{"content":"Initializes a new instance of the <bpt id=\"p1\">&lt;xref href=\"System.Xaml.XamlException\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept> class.","pos":[0,87],"source":"Initializes a new instance of the <xref href=\"System.Xaml.XamlException\"></xref> class."},{"content":"The instance contains a specified error message, inner exception, and line information.","pos":[88,175]}]},{"pos":[7175,7330],"content":"The message that describes the exception. The caller of this constructor must make sure that this string has been localized for the current system culture.","needQuote":true,"needEscape":true,"nodes":[{"content":"The message that describes the exception. The caller of this constructor must make sure that this string has been localized for the current system culture.","pos":[0,155],"nodes":[{"content":"The message that describes the exception.","pos":[0,41]},{"content":"The caller of this constructor must make sure that this string has been localized for the current system culture.","pos":[42,155]}]}]},{"pos":[7404,7461],"content":"The exception that is the cause of the current exception.","needQuote":true,"needEscape":true,"nodes":[{"content":"The exception that is the cause of the current exception.","pos":[0,57]}]},{"pos":[7527,7599],"content":"The line number to report to debugging or to line information consumers.","needQuote":true,"needEscape":true,"nodes":[{"content":"The line number to report to debugging or to line information consumers.","pos":[0,72]}]},{"pos":[7667,7738],"content":"The line position to report to debugging or line information consumers.","needQuote":true,"needEscape":true,"nodes":[{"content":"The line position to report to debugging or line information consumers.","pos":[0,71]}]},{"pos":[8442,8583],"content":"Implements <xref:System.Runtime.Serialization.ISerializable.GetObjectData*> and provides serialization support for the line information data.","needQuote":true,"needEscape":true,"nodes":[{"content":"Implements &lt;xref:System.Runtime.Serialization.ISerializable.GetObjectData*&gt; and provides serialization support for the line information data.","pos":[0,141],"source":"Implements <xref:System.Runtime.Serialization.ISerializable.GetObjectData*> and provides serialization support for the line information data."}]},{"pos":[8861,8910],"content":"The object that holds the serialized object data.","needQuote":true,"needEscape":true,"nodes":[{"content":"The object that holds the serialized object data.","pos":[0,49]}]},{"pos":[9006,9065],"content":"The contextual information about the source or destination.","needQuote":true,"needEscape":true,"nodes":[{"content":"The contextual information about the source or destination.","pos":[0,59]}]},{"pos":[9235,9317],"content":"<ph id=\"ph1\">&lt;code&gt;info&lt;/code&gt;</ph> is <bpt id=\"p1\">&lt;xref uid=\"langword_csharp_null\" name=\"null\" href=\"\"&gt;</bpt><ept id=\"p1\">&lt;/xref&gt;</ept>.","needQuote":true,"needEscape":true,"source":"<code>info</code> is <xref uid=\"langword_csharp_null\" name=\"null\" href=\"\"></xref>."},{"pos":[9650,9746],"content":"Gets or sets the line number component of XAML text line information that the exception reports.","needQuote":true,"needEscape":true,"nodes":[{"content":"Gets or sets the line number component of XAML text line information that the exception reports.","pos":[0,96]}]},{"pos":[9872,9932],"content":"The line number component of the XAML text line information.","needQuote":true,"needEscape":true,"nodes":[{"content":"The line number component of the XAML text line information.","pos":[0,60]}]},{"pos":[10342,10440],"content":"Gets or sets the line position component of XAML text line information that the exception reports.","needQuote":true,"needEscape":true,"nodes":[{"content":"Gets or sets the line position component of XAML text line information that the exception reports.","pos":[0,98]}]},{"pos":[10568,10626],"content":"The line position component of XAML text line information.","needQuote":true,"needEscape":true,"nodes":[{"content":"The line position component of XAML text line information.","pos":[0,58]}]},{"pos":[11013,11131],"content":"Gets or sets the exception message, and if line information is available, appends the line information to the message.","needQuote":true,"needEscape":true,"nodes":[{"content":"Gets or sets the exception message, and if line information is available, appends the line information to the message.","pos":[0,118]}]},{"pos":[11143,11501],"content":"<xref:System.Xaml.XamlException> overrides this property so that line information is directly included in the message and is therefore more easily visible for debuggers or other exception consumers. If line information is available, this property includes the line information. If line information is unavailable, the message is the same as the base message.","needQuote":true,"needEscape":true,"extradata":"MT","nodes":[{"content":"<xref:System.Xaml.XamlException> overrides this property so that line information is directly included in the message and is therefore more easily visible for debuggers or other exception consumers. If line information is available, this property includes the line information. If line information is unavailable, the message is the same as the base message.","pos":[0,358],"nodes":[{"content":"&lt;xref:System.Xaml.XamlException&gt; overrides this property so that line information is directly included in the message and is therefore more easily visible for debuggers or other exception consumers.","pos":[0,198],"source":"<xref:System.Xaml.XamlException> overrides this property so that line information is directly included in the message and is therefore more easily visible for debuggers or other exception consumers."},{"content":"If line information is available, this property includes the line information.","pos":[199,277]},{"content":"If line information is unavailable, the message is the same as the base message.","pos":[278,358]}]}]},{"pos":[11622,11688],"content":"The exception message that includes the appended line information.","needQuote":true,"needEscape":true,"nodes":[{"content":"The exception message that includes the appended line information.","pos":[0,66]}]}],"content":"### YamlMime:ManagedReference\nitems:\n- uid: System.Xaml.XamlException\n  id: XamlException\n  children:\n  - System.Xaml.XamlException.#ctor\n  - System.Xaml.XamlException.#ctor(System.String)\n  - System.Xaml.XamlException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  - System.Xaml.XamlException.#ctor(System.String,System.Exception)\n  - System.Xaml.XamlException.#ctor(System.String,System.Exception,System.Int32,System.Int32)\n  - System.Xaml.XamlException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  - System.Xaml.XamlException.LineNumber\n  - System.Xaml.XamlException.LinePosition\n  - System.Xaml.XamlException.Message\n  langs:\n  - csharp\n  name: XamlException\n  nameWithType: XamlException\n  fullName: System.Xaml.XamlException\n  type: Class\n  summary: The exception that is thrown for a general XAML reader or XAML writer exception. See Remarks.\n  remarks: More specific exceptions such as <xref:System.Xaml.XamlParseException> exist and are often thrown instead of XamlException. Specific exceptions typically provide more precise exception information as properties of the exception, or just because the specific exception was raised.\n  syntax:\n    content: 'public class XamlException : Exception'\n  inheritance:\n  - System.Exception\n  implements: []\n  inheritedMembers: []\n  platform:\n  - net462\n- uid: System.Xaml.XamlException.#ctor\n  id: '#ctor'\n  parent: System.Xaml.XamlException\n  langs:\n  - csharp\n  name: XamlException()\n  nameWithType: XamlException.XamlException()\n  fullName: System.Xaml.XamlException.XamlException()\n  type: Constructor\n  assemblies:\n  - System.Xaml\n  namespace: System.Xaml\n  summary: Initializes a new instance of the <xref href=\"System.Xaml.XamlException\"></xref> class. The instance contains a system-supplied message that describes the error.\n  syntax:\n    content: public XamlException ();\n    parameters: []\n  overload: System.Xaml.XamlException.#ctor*\n  exceptions: []\n  platform:\n  - net462\n- uid: System.Xaml.XamlException.#ctor(System.String)\n  id: '#ctor(System.String)'\n  parent: System.Xaml.XamlException\n  langs:\n  - csharp\n  name: XamlException(String)\n  nameWithType: XamlException.XamlException(String)\n  fullName: System.Xaml.XamlException.XamlException(String)\n  type: Constructor\n  assemblies:\n  - System.Xaml\n  namespace: System.Xaml\n  summary: Initializes a new instance of the <xref href=\"System.Xaml.XamlException\"></xref> class. The instance contains a specified message that describes the error.\n  syntax:\n    content: public XamlException (string message);\n    parameters:\n    - id: message\n      type: System.String\n      description: The message that describes the exception. The caller of this constructor must make sure that this string has been localized for the current system culture.\n  overload: System.Xaml.XamlException.#ctor*\n  exceptions: []\n  platform:\n  - net462\n- uid: System.Xaml.XamlException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  id: '#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)'\n  parent: System.Xaml.XamlException\n  langs:\n  - csharp\n  name: XamlException(SerializationInfo,StreamingContext)\n  nameWithType: XamlException.XamlException(SerializationInfo,StreamingContext)\n  fullName: System.Xaml.XamlException.XamlException(SerializationInfo,StreamingContext)\n  type: Constructor\n  assemblies:\n  - System.Xaml\n  namespace: System.Xaml\n  summary: Initializes a new instance of the <xref href=\"System.Xaml.XamlException\"></xref> class with serialized data.\n  syntax:\n    content: protected XamlException (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);\n    parameters:\n    - id: info\n      type: System.Runtime.Serialization.SerializationInfo\n      description: The object that holds the serialized object data.\n    - id: context\n      type: System.Runtime.Serialization.StreamingContext\n      description: The contextual information about the source or destination.\n  overload: System.Xaml.XamlException.#ctor*\n  exceptions:\n  - type: System.ArgumentNullException\n    commentId: T:System.ArgumentNullException\n    description: <code>info</code> is <xref uid=\"langword_csharp_null\" name=\"null\" href=\"\"></xref>.\n  platform:\n  - net462\n- uid: System.Xaml.XamlException.#ctor(System.String,System.Exception)\n  id: '#ctor(System.String,System.Exception)'\n  parent: System.Xaml.XamlException\n  langs:\n  - csharp\n  name: XamlException(String,Exception)\n  nameWithType: XamlException.XamlException(String,Exception)\n  fullName: System.Xaml.XamlException.XamlException(String,Exception)\n  type: Constructor\n  assemblies:\n  - System.Xaml\n  namespace: System.Xaml\n  summary: Initializes a new instance of the <xref href=\"System.Xaml.XamlException\"></xref> class. The instance contains a specified error message and a reference to the inner exception that is the cause of this exception.\n  remarks: An exception that is thrown because a previous exception should include a reference to the previous exception in the <xref:System.Exception.InnerException%2A> property. The <xref:System.Exception.InnerException%2A> property returns the same value that is passed into the constructor; or it returns `null`if the <xref:System.Exception.InnerException%2A> property does not provide the inner exception value to the constructor.\n  syntax:\n    content: public XamlException (string message, Exception innerException);\n    parameters:\n    - id: message\n      type: System.String\n      description: The message that describes the exception. The caller of this constructor must make sure that this string has been localized for the current system culture.\n    - id: innerException\n      type: System.Exception\n      description: The exception that is the cause of the current exception. If the `innerException` parameter is not <xref uid=\"langword_csharp_null\" name=\"null\" href=\"\"></xref>, the current exception is raised in a <xref uid=\"langword_csharp_catch\" name=\"catch\" href=\"\"></xref> block that handles the inner exception.\n  overload: System.Xaml.XamlException.#ctor*\n  exceptions: []\n  platform:\n  - net462\n- uid: System.Xaml.XamlException.#ctor(System.String,System.Exception,System.Int32,System.Int32)\n  id: '#ctor(System.String,System.Exception,System.Int32,System.Int32)'\n  parent: System.Xaml.XamlException\n  langs:\n  - csharp\n  name: XamlException(String,Exception,Int32,Int32)\n  nameWithType: XamlException.XamlException(String,Exception,Int32,Int32)\n  fullName: System.Xaml.XamlException.XamlException(String,Exception,Int32,Int32)\n  type: Constructor\n  assemblies:\n  - System.Xaml\n  namespace: System.Xaml\n  summary: Initializes a new instance of the <xref href=\"System.Xaml.XamlException\"></xref> class. The instance contains a specified error message, inner exception, and line information.\n  syntax:\n    content: public XamlException (string message, Exception innerException, int lineNumber, int linePosition);\n    parameters:\n    - id: message\n      type: System.String\n      description: The message that describes the exception. The caller of this constructor must make sure that this string has been localized for the current system culture.\n    - id: innerException\n      type: System.Exception\n      description: The exception that is the cause of the current exception.\n    - id: lineNumber\n      type: System.Int32\n      description: The line number to report to debugging or to line information consumers.\n    - id: linePosition\n      type: System.Int32\n      description: The line position to report to debugging or line information consumers.\n  overload: System.Xaml.XamlException.#ctor*\n  exceptions: []\n  platform:\n  - net462\n- uid: System.Xaml.XamlException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  id: GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  parent: System.Xaml.XamlException\n  langs:\n  - csharp\n  name: GetObjectData(SerializationInfo,StreamingContext)\n  nameWithType: XamlException.GetObjectData(SerializationInfo,StreamingContext)\n  fullName: System.Xaml.XamlException.GetObjectData(SerializationInfo,StreamingContext)\n  type: Method\n  assemblies:\n  - System.Xaml\n  namespace: System.Xaml\n  summary: Implements <xref:System.Runtime.Serialization.ISerializable.GetObjectData*> and provides serialization support for the line information data.\n  syntax:\n    content: public override void GetObjectData (System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context);\n    parameters:\n    - id: info\n      type: System.Runtime.Serialization.SerializationInfo\n      description: The object that holds the serialized object data.\n    - id: context\n      type: System.Runtime.Serialization.StreamingContext\n      description: The contextual information about the source or destination.\n  overload: System.Xaml.XamlException.GetObjectData*\n  exceptions:\n  - type: System.ArgumentNullException\n    commentId: T:System.ArgumentNullException\n    description: <code>info</code> is <xref uid=\"langword_csharp_null\" name=\"null\" href=\"\"></xref>.\n  platform:\n  - net462\n- uid: System.Xaml.XamlException.LineNumber\n  id: LineNumber\n  parent: System.Xaml.XamlException\n  langs:\n  - csharp\n  name: LineNumber\n  nameWithType: XamlException.LineNumber\n  fullName: System.Xaml.XamlException.LineNumber\n  type: Property\n  assemblies:\n  - System.Xaml\n  namespace: System.Xaml\n  summary: Gets or sets the line number component of XAML text line information that the exception reports.\n  syntax:\n    content: public int LineNumber { get; protected set; }\n    return:\n      type: System.Int32\n      description: The line number component of the XAML text line information.\n  overload: System.Xaml.XamlException.LineNumber*\n  exceptions: []\n  platform:\n  - net462\n- uid: System.Xaml.XamlException.LinePosition\n  id: LinePosition\n  parent: System.Xaml.XamlException\n  langs:\n  - csharp\n  name: LinePosition\n  nameWithType: XamlException.LinePosition\n  fullName: System.Xaml.XamlException.LinePosition\n  type: Property\n  assemblies:\n  - System.Xaml\n  namespace: System.Xaml\n  summary: Gets or sets the line position component of XAML text line information that the exception reports.\n  syntax:\n    content: public int LinePosition { get; protected set; }\n    return:\n      type: System.Int32\n      description: The line position component of XAML text line information.\n  overload: System.Xaml.XamlException.LinePosition*\n  exceptions: []\n  platform:\n  - net462\n- uid: System.Xaml.XamlException.Message\n  id: Message\n  parent: System.Xaml.XamlException\n  langs:\n  - csharp\n  name: Message\n  nameWithType: XamlException.Message\n  fullName: System.Xaml.XamlException.Message\n  type: Property\n  assemblies:\n  - System.Xaml\n  namespace: System.Xaml\n  summary: Gets or sets the exception message, and if line information is available, appends the line information to the message.\n  remarks: <xref:System.Xaml.XamlException> overrides this property so that line information is directly included in the message and is therefore more easily visible for debuggers or other exception consumers. If line information is available, this property includes the line information. If line information is unavailable, the message is the same as the base message.\n  syntax:\n    content: public override string Message { get; }\n    return:\n      type: System.String\n      description: The exception message that includes the appended line information.\n  overload: System.Xaml.XamlException.Message*\n  exceptions: []\n  platform:\n  - net462\nreferences:\n- uid: System.Exception\n  isExternal: false\n  name: System.Exception\n- uid: System.ArgumentNullException\n  isExternal: true\n  name: System.ArgumentNullException\n- uid: System.Xaml.XamlException.#ctor\n  parent: System.Xaml.XamlException\n  isExternal: false\n  name: XamlException()\n  nameWithType: XamlException.XamlException()\n  fullName: System.Xaml.XamlException.XamlException()\n- uid: System.Xaml.XamlException.#ctor(System.String)\n  parent: System.Xaml.XamlException\n  isExternal: false\n  name: XamlException(String)\n  nameWithType: XamlException.XamlException(String)\n  fullName: System.Xaml.XamlException.XamlException(String)\n- uid: System.String\n  parent: System\n  isExternal: true\n  name: String\n  nameWithType: String\n  fullName: System.String\n- uid: System.Xaml.XamlException.#ctor(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  parent: System.Xaml.XamlException\n  isExternal: false\n  name: XamlException(SerializationInfo,StreamingContext)\n  nameWithType: XamlException.XamlException(SerializationInfo,StreamingContext)\n  fullName: System.Xaml.XamlException.XamlException(SerializationInfo,StreamingContext)\n- uid: System.Runtime.Serialization.SerializationInfo\n  parent: System.Runtime.Serialization\n  isExternal: false\n  name: SerializationInfo\n  nameWithType: SerializationInfo\n  fullName: System.Runtime.Serialization.SerializationInfo\n- uid: System.Runtime.Serialization.StreamingContext\n  parent: System.Runtime.Serialization\n  isExternal: true\n  name: StreamingContext\n  nameWithType: StreamingContext\n  fullName: System.Runtime.Serialization.StreamingContext\n- uid: System.Xaml.XamlException.#ctor(System.String,System.Exception)\n  parent: System.Xaml.XamlException\n  isExternal: false\n  name: XamlException(String,Exception)\n  nameWithType: XamlException.XamlException(String,Exception)\n  fullName: System.Xaml.XamlException.XamlException(String,Exception)\n- uid: System.Xaml.XamlException.#ctor(System.String,System.Exception,System.Int32,System.Int32)\n  parent: System.Xaml.XamlException\n  isExternal: false\n  name: XamlException(String,Exception,Int32,Int32)\n  nameWithType: XamlException.XamlException(String,Exception,Int32,Int32)\n  fullName: System.Xaml.XamlException.XamlException(String,Exception,Int32,Int32)\n- uid: System.Int32\n  parent: System\n  isExternal: true\n  name: Int32\n  nameWithType: Int32\n  fullName: System.Int32\n- uid: System.Xaml.XamlException.GetObjectData(System.Runtime.Serialization.SerializationInfo,System.Runtime.Serialization.StreamingContext)\n  parent: System.Xaml.XamlException\n  isExternal: false\n  name: GetObjectData(SerializationInfo,StreamingContext)\n  nameWithType: XamlException.GetObjectData(SerializationInfo,StreamingContext)\n  fullName: System.Xaml.XamlException.GetObjectData(SerializationInfo,StreamingContext)\n- uid: System.Xaml.XamlException.LineNumber\n  parent: System.Xaml.XamlException\n  isExternal: false\n  name: LineNumber\n  nameWithType: XamlException.LineNumber\n  fullName: System.Xaml.XamlException.LineNumber\n- uid: System.Xaml.XamlException.LinePosition\n  parent: System.Xaml.XamlException\n  isExternal: false\n  name: LinePosition\n  nameWithType: XamlException.LinePosition\n  fullName: System.Xaml.XamlException.LinePosition\n- uid: System.Xaml.XamlException.Message\n  parent: System.Xaml.XamlException\n  isExternal: false\n  name: Message\n  nameWithType: XamlException.Message\n  fullName: System.Xaml.XamlException.Message\n- uid: System.Xaml.XamlException.#ctor*\n  parent: System.Xaml.XamlException\n  isExternal: false\n  name: XamlException\n  nameWithType: XamlException.XamlException\n- uid: System.Xaml.XamlException.GetObjectData*\n  parent: System.Xaml.XamlException\n  isExternal: false\n  name: GetObjectData\n  nameWithType: XamlException.GetObjectData\n- uid: System.Xaml.XamlException.LineNumber*\n  parent: System.Xaml.XamlException\n  isExternal: false\n  name: LineNumber\n  nameWithType: XamlException.LineNumber\n- uid: System.Xaml.XamlException.LinePosition*\n  parent: System.Xaml.XamlException\n  isExternal: false\n  name: LinePosition\n  nameWithType: XamlException.LinePosition\n- uid: System.Xaml.XamlException.Message*\n  parent: System.Xaml.XamlException\n  isExternal: false\n  name: Message\n  nameWithType: XamlException.Message\n"}